LeftPad/logic.rb

```diff
@@ -0,0 +1,15 @@
+class Left_Pad
+  def left_pad(string, times, char, bool)
```

Comment:
bool is not really a descriptive name for a parameter. Maybe something more intention revealing, like `reversed` would be a good idea :)